src/bootstrap/builder.rs

```diff
@@ -434,9 +434,13 @@ impl<'a> Builder<'a> {
                 test::RustdocJson,
                 // Run bootstrap close to the end as it's unlikely to fail
                 test::Bootstrap,
+                test::CBMC,
```

Comment:
Can you add a comment here to mark the beginning of RMC tests?

Reply:
Done!